docs/BASE_SETUP.md

```diff
@@ -5,9 +5,12 @@
 - Legal entities (under the root legal entity `C000000`) per legal entity entry with users array in the file [legal-entities-with-users.json](../src/main/resources/data/legal-entities-with-users.json) - configurable, see section *Custom data*
 
 For legal entities and users in the file [legal-entities-with-users.json](../src/main/resources/data/legal-entities-with-users.json):
-- All job profiles and product groups are assigned to the users via master service agreement of the legal entities from the input file.
-- Job profiles as defined in [job-profiles.json](../src/main/resources/data/job-profiles.json)
-- Product groups as defined in [product-group-seed.json](../src/main/resources/data/seed/product-group-seed.json)
+- All job profiles are assigned to the users via master service agreement of the legal entities from the input file.
+    - Job profiles are defined in [job-profiles.json](../src/main/resources/data/job-profiles.json)
+    - You can limit the job profiles assigned to a user by explicitly referring to one of the roles defined in the job profile. P.e. the `retail` role on a user will result in only assignment of the job profile `Retail User`.   
+- All product groups are assigned to the users via master service agreement of the legal entities from the input file.
+    - Product groups are defined in [product-group-seed.json](../src/main/resources/data/seed/product-group-seed.json)
+    - You can limit product groups assigned to a user by explicitly referring to the actual names in productGroupNames of a user. P.e. a user with only one productGroupName `Payroll` will only be granted permissions for the `Payroll` product group.
```

Comment:
P.e.? E.g.?

Reply:
good one, got that mixed up (per exemplum)